### ISSUE TITLE
Is this plugin deprecated?

### ISSUE THREAD

**Issue by michielbdejong** [sharafian/ilp-plugin-xrp-asym#9]
It looks like it was split into https://github.com/interledgerjs/ilp-plugin-xrp-asym-server and https://github.com/interledgerjs/ilp-plugin-xrp-asym-client?

**Comment by sharafian** [comment]
Yeah, all I'm keeping here are some tests that use both plugins at once
